app/models/plot.rb

```diff
@@ -1,3 +1,4 @@
 class Plot < ApplicationRecord
    belongs_to :garden
+   has_many :plants
```

Comment:
Instead:
`has_many :plot_plants`
`has_many :plants, through: :plot_plants`

You'd also need a model for the PlotPlant (joins record)